components/chat/chatBar/search/catalog.tsx

```diff
@@ -0,0 +1,545 @@
+    },
+    {
+      name: 'Image Generation',
+      description: 'Allows the assistant to generate images.',
+      url: 'github.com/gptscript-ai/dalle-image-generation@gateway',
+      tags: ['images', 'ai', 'generation'],
+      icon: <FaPaintBrush className="text-lg" />,
+    },
+    {
+      name: 'Search The Internet',
+      description: 'Allows the assistant to search the web for answers.',
+      url: 'github.com/gptscript-ai/answers-from-the-internet',
+      tags: ['search', 'web', 'internet'],
+      icon: <GoGlobe className="text-lg" />,
+    },
+    {
+      name: 'Crawl Website',
+      description: 'Allows the assistant to search a website.',
+      url: 'github.com/gptscript-ai/search-website',
+      tags: ['search', 'web', 'site'],
+      icon: <GoSearch className="text-lg" />,
+    },
+    {
+      name: 'Browser',
+      description:
+        'Provides the assistant with the ability to interact with the web via a Chrome window.',
+      url: 'github.com/gptscript-ai/browser',
+      tags: ['browser', 'web', 'chrome', 'search'],
+      icon: <GoBrowser className="text-lg" />,
+    },
+  ],
+  Productivity: [
+    {
+      name: 'Slack',
+      description: 'Allows the assistant to interact with Slack.',
+      url: 'github.com/gptscript-ai/tools/apis/slack/write',
+      tags: ['slack', 'messaging', 'teams', 'api'],
+      icon: <AiOutlineSlack className="text-lg" />,
+    },
+    {
+      name: 'Notion',
+      description: 'Allows the assistant to interact with Notion.',
+      url: 'github.com/gptscript-ai/tools/apis/notion/write',
+      tags: ['notion', 'documentation', 'notes', 'api'],
+      icon: <SiNotion className="text-lg" />,
+    },
+    // {
+    //     name: 'Trello',
+    //     description: 'Allows the assistant to interact with Trello.',
+    //     url: 'github.com/gptscript-ai/tools/apis/trello',
+    //     tags: ['trello', 'project', 'management', 'api'],
+    //     icon: <FaTrello className="text-lg" />,
+    // },
+    {
+      name: 'Hubspot',
+      description: 'Allows the assistant to interact with Hubspot.',
+      url: 'github.com/gptscript-ai/tools/apis/hubspot/crm',
+      tags: ['hubspot', 'api'],
+      icon: <FaHubspot className="text-lg" />,
+    },
+    {
+      name: 'Outlook Mail',
+      description:
+        'Allows the assistant to send and receive emails via Outlook.',
+      url: 'github.com/gptscript-ai/tools/apis/outlook/mail/manage',
+      tags: ['email', 'office', 'microsoft', 'service'],
+      icon: <PiMicrosoftOutlookLogoDuotone className="text-lg" />,
+    },
+    {
+      name: 'Outlook Calendar',
+      description: 'Allows the assistant to interact with Outlook Calendar.',
+      url: 'github.com/gptscript-ai/tools/apis/outlook/calendar/manage',
+      tags: ['calendar', 'office', 'microsoft', 'service'],
+      icon: <PiMicrosoftOutlookLogoDuotone className="text-lg" />,
+    },
+  ],
+  'Working with Local Files': [
+    // {
```

Comment:
Any reason to comment out instead of removing?

Reply:
these items could be added back in, didn't want to create that much typing work if that was the case

Reply:
@ryanhopperlowe - I'm with @StrongMonkey here - go ahead and delete these. It is easy enough to pull these out of git history when and if we want to restore them.

I'll have to admit: commented out code is a bit of a pet peeve of mine.